fetcher/fetcher.go

```diff
@@ -86,12 +99,14 @@ func (bot *fetcherImpl) catchupLoop() {
 
 		blockbytes, err = aclient.BlockRaw(bot.nextRound).Do(context.Background())
 		if err != nil {
+			bot.err = err
```

Comment:
Looks like bot.err is never cleared. Which of these errors recoverable? If normal progress resumes we should clear bot.err.

Reply:
Good idea, each time we finish processing a block I clear this field now.